apps/script-service/Services/IScriptFunctions.cs

```diff
@@ -0,0 +1,18 @@
+
+using Adsp.Platform.ScriptService.Services.Platform;
+using Adsp.Platform.ScriptService.Services.Util;
+using Adsp.Sdk;
+using NLua;
```

Comment:
I think there are some unused imported namespaces here

Reply:
Hi @tzuge, the unused namespaces have been removed. Thank you!